src/data/CreateWallet.ts

```diff
@@ -1,18 +1,20 @@
 // This content is used as a code example in /src/pages/index.js
 // Kept here for easy formatting.
```

Comment:
> @samajammin from the comment I suspect that we shouldn't migrate this file to TS. Or maybe we can just delete it since it is not being used by other files. Let us know if you want to keep it as it is.
> 
> 
> 
> Sorry @vdusart for not detecting this question before. I know this file was on the list. Lets wait for Sam's confirmation before merging this 👍🏼 

Yes sure, no worries about that 😁

Reply:
You can find exactly the content of the `CreateWallet.ts` file on [this page](https://ethereum.org/en/) inside the `A JavaScript Ethereum wallet` code example, but it comes from [this file](https://github.com/ethereum/ethereum-org-website/blob/dev/src/pages/index.tsx#L603-L650)...

Reply:
That is right. From a conversation with Sam, let's keep this file in `.js` format as the plan is to import it in the code example block that you mentioned. See #7150 for more context.

Reply:
Oh ok !
I'm gonna revert the changes on this file then and do the refactor to import the files :smile:

Reply:
Ok, cool. If possible lets do that refactor in a different PR so that we can merge this one earlier and to have things separated. What do you think?

Reply:
Yes of course, it doesn't really make sense to make the changes here. I asked sam to assign me the issue #7150 .

Reply:
Perfect! thanks